docs-compose/overwatch_docs/content/DeployOverwatch/CloudInfra/AWS.md

```diff
@@ -26,11 +26,19 @@ There are two primary sources of data that need to be configured:
     * Best Practice - Multi-Workspace -- When multiple workspaces are using Overwatch within a single region it's best to
       ensure that each are going to their own prefix, even if sharing a bucket. This greatly reduces Overwatch scan times
       as the log files build up.
-    * Best Practice - Sharding -- When many clusters send their logs to the same prefix it reduces the AWS read parallelism
-      for these small files. It's best to find a methodology for log prefixes that works for your org. One example would
-      be to have each workspace and each team store their logs in a specific prefix, etc to break up the number of log
-      files buried inside a single prefix. Overwatch will do all the work to aggregate these logs and deliver the data
-      to the data model.
+* Best Practice - Sharding -- When many clusters send their logs to the same prefix it reduces the AWS read parallelism
```

Comment:
dropping this section because each cluster_id creates a new folder and AWS will redistribute this data since it's sharded by cluster_id folders already. You're right but I'm not sure this belongs here -- I think it will just add unnecessary confusion.